app-backend/batch/src/main/scala/stacExport/WriteStacCatalog.scala

```diff
@@ -56,7 +55,16 @@ final case class WriteStacCatalog(exportId: UUID)(
     )
 
     val sceneItems = sceneTaskAnnotation.scenes flatMap { scene =>
```

Comment:
This is kind of nitpicky, but changing this to a tuple hurts readability in my opinion with the `sceneItems map { _._1 }` call below.

Reply:
Ah yeah that's fair -- I'll promote it to a case class

Reply:
It's now a case class with `item` and `ingestLocation` fields 3fb9f3e